releasenotes/2021-March-19.md

```diff
@@ -0,0 +1,33 @@
+This release notes contain new Events that we have released and some changes to existing event.
+
+# New Events Added
+Azure Communication Services added the following new events to Chat functionality:
+
+| Event type                                                  | Description                                                                                    |
+| ----------------------------------------------------------- | ---------------------------------------------------------------------------------------------- |
+| Microsoft.Communication.ChatParticipantAddedToThreadWithUser|  Published for a user when a new  participant is added to a chat thread, that the user is part of.|
+| Microsoft.Communication.ChatParticipantRemovedFromThreadWithUser |  Published for a user when a participant is removed from a chat thread, that the user is part of. |
+| Microsoft.Communication.ChatThreadCreated  | Published when a chat thread is created  |
+| Microsoft.Communication.ChatThreadDeleted| Published when a chat thread is deleted  |
+| Microsoft.Communication.ChatThreadParticipantAdded | Published when a new participant is added to a chat thread  |
+| Microsoft.Communication.ChatThreadParticipantRemoved | Published when a new participant is added to a chat thread.  |  
+| Microsoft.Communication.ChatMessageReceivedInThread | Published when a message is received in a chat thread  |    
+| Microsoft.Communication.ChatThreadPropertiesUpdated| Published when a chat thread's properties like topic are updated.|    
+| Microsoft.Communication.ChatMessageEditedInThread | Published when a message is edited in a chat thread |  
+| Microsoft.Communication.ChatMessageDeletedInThread | Published when a message is deleted in  a chat thread  | 
+
+# Breaking Changes to Existing Events 
+Azure Communication Services has chagnes to the following events. With this changes, for Customers using already shipped SDKs will continue functioning without trouble because there is no change to the existing events and properties, and new changes will introduce  duplicates events (ChatMemberAddedToThreadWithUser;
+ChatMemberRemovedFromThreadWithUser) and properties for 90 days and these changes will be deprecated after 90 Days. 
```

Comment:
How about re-wording this to:

"These changes will not impact the functioning of already shipped SDKs since Azure Communication Services will continue to dual publish the events/properties until the time they are officially deprecated. Our standard deprecation timeline is 90 days, so we expect all customers to switch to the updated SDKs by that timeline. At the end of the 90 days, the following events/properties will be deprecated:

<List of events/properties to be deprecated with details of the replacement event/property>"